cmake/HeaderOnlyLibrary.cmake

```diff
@@ -43,7 +43,7 @@ function(header_only_library)
     ${ARG_NAME}
     INTERFACE
     FILE_SET HEADERS
-    BASE_DIRS "${PROJECT_SOURCE_DIR}/au/code"
+    BASE_DIRS "${PROJECT_SOURCE_DIR}"
```

Comment:
Maybe for some reason I had thought (in #266) that we shouldn't set `BASE_DIRS` to `${PROJECT_SOURCE_DIR}`?

Maybe because `cmake` would then be a subfolder of the project source, including `cmake/build`, where the build lives?

I don't _think_ that's actually a problem.  I'm just doing my best to try to remember why we didn't simply do this in the first place.